packages/mint-components/tsconfig.json

```diff
@@ -16,7 +16,7 @@
     "moduleResolution": "node",
     "module": "esnext",
     "target": "es2019",
-    "noUnusedLocals": true,
+    "noUnusedLocals": false,
```

Comment:
Why?

Reply:
I'm not sure when this was added... i'm guessing it was a shortcut for demo hooks where the props are usually unused